projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -2902,12 +2848,10 @@ void outputAnswerWithLog(Answer answer) {
                prettyAnswerString);
       }
       catch (Exception e) {
-         BatfishException be = new BatfishException(
-               "Error in sending answer",
+         BatfishException be = new BatfishException("Error in sending answer",
```

Comment:
it feels like we're breaking as late as possible rather than semantically. I'd probably actually write

```java
BatfishException be = 
      new BatfishException("Error in sending answer", e);
```

for maximum clarity, assuming it can't be a 1-linear.